### ISSUE TITLE
Unable to play YouTube Videos

### ISSUE THREAD

**Issue by slannon97** [gumerov-amir/TTMediaBot#123]
The bot is unable to play YouTube videos. I updated to the latest version, updated the SDK to 5.15, and ran pip3 install --upgrade yt-dlp to upgrade YouTube-DLP. The player refuses to play or search for videos. I get messages in the terminal that say I need to log in to verify I'm not a bot. Is this possible in the current version or does something need to be implemented in the future?

**Comment by cyrmax** [comment]
This is not an issue of TTMediaBot. This happens because Youtube implements more strict protection against bots and automated requests. As far as I know, nothing can be done with it, only looking for new IP addresses which are not banned by Youtube and Google. But beware that almost any VDS IP address will be blacklisted because Youtube knows about most hosting providers and their subnets.

**Comment by slannon97** [comment]
That's what I thought. Thanks for the confirmation. I'll most likely stop using the bot then, since YouTube was the most useful feature. I'll leave this issue open in case someone else wants to comment or comes up with a fix, but it doesn't seem likely at this point.

**Comment by Andi593** [comment]
Yes, correctly. It is a problem of Google, but TTMediaBot needs an implementation that allows to sign in on a Google account, using Oauth2 to solve this problem.
